src/testing/test-utils.test.ts

```diff
@@ -27,9 +27,11 @@ describe('SelectHelper', () => {
     // Act
     const select = new SelectHelper(screen.getByLabelText('Test Select'), user);
     const options = await select.getOptions();
+    const selectedOptions = select.getSelectedOptions();
```

Comment:
Can we add a dedicated test for this method? One that also includes a positive case. Like render the Select, check that `getSelectedOptions` returns an empty list, select an option, check that `getSelectedOptions` returns the option just selected.